linux/x86-64/python3/setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 setup(
     name="pyacrcloud",
-    version="1.0.0",
+    version="1.0.1",
```

Comment:
For all of these versions, instead of bumping the microversion we should do, eg: "1.0.0.post1".  If/when we submit a PR upstream we can bump the versions then, but I'm going to recommend that we don't submit this upstream now.